taxi/views.py

```diff
@@ -1,3 +1,20 @@
+from django.http import HttpResponse
 from django.shortcuts import render
+from taxi.models import Manufacturer, Driver, Car
 
-# Create your views here.
+
+def homeland(request):
+    return HttpResponse(f"i am the man")
```

Comment:
You are. But maybe not neccessary here;)

Reply:
was just testing how it works)